.github/workflows/ci-windows-latest.yaml

```diff
@@ -0,0 +1,133 @@
+
```

Comment:
Should it be better for the CI to take in consideration all the versions? so also 3.6.12, 3.6.9, 3.8.16, 3.9.8?